src/constraints/outage_constraints.jl

```diff
@@ -143,14 +154,73 @@ function add_MG_fuel_burn_constraints(m,p)
+  
+    # Conditionally add dvFuelBurnYIntercept if coefficient p.FuelBurnYIntRate is greater than ~zero
+    if fuel_burn_intercept > 1.0E-7
+        #Constraint (1c1): Total Fuel burn for CHP **with** y-intercept fuel burn and supplementary firing
+        @constraint(m, MGCHPFuelBurnCon[t in p.techs.chp, s in p.s.electric_utility.scenarios, tz in p.s.electric_utility.outage_start_time_steps],
+            m[Symbol("dvMGFuelUsed"*_n)][t,s,tz]  == p.hours_per_time_step * 
+                m[Symbol("dvMGCHPFuelBurnYIntercept"*_n)][s,tz] +
+                sum(p.production_factor[t,tz+ts-1] * fuel_burn_slope * m[Symbol("dvMGRatedProduction"*_n)][t,s,tz,ts]
+                    for ts in 1:p.s.electric_utility.outage_durations[s])
+        )
+
+        #Constraint (1d): Y-intercept fuel burn for CHP across the scenario outage time steps
+        @constraint(m, MGCHPFuelBurnYIntCon[t in p.techs.chp, s in p.s.electric_utility.scenarios, tz in p.s.electric_utility.outage_start_time_steps],
+            m[Symbol("binMGCHPIsOnInTS"*_n)][s,tz,ts] => 
+                {m[Symbol("dvMGCHPFuelBurnYIntercept"*_n)][s,tz] >= sum(fuel_burn_intercept * m[Symbol("dvMGsize"*_n)][t] 
+                    for _ in 1:p.s.electric_utility.outage_durations[s])}
+        )
+    else
+        #Constraint (1c2): Total Fuel burn for CHP **without** y-intercept fuel burn
+        @constraint(m, MGCHPFuelBurnConLinear[t in p.techs.chp, s in p.s.electric_utility.scenarios, tz in p.s.electric_utility.outage_start_time_steps],
+            m[Symbol("dvMGFuelUsed"*_n)][t,s,tz]  == p.hours_per_time_step *
+                sum(p.production_factor[t,tz+ts-1] * fuel_burn_slope * m[Symbol("dvMGRatedProduction"*_n)][t,s,tz,ts]
+                for ts in 1:p.s.electric_utility.outage_durations[s])
+        )
+    end 
+    
+    @constraint(m, [s in p.s.electric_utility.scenarios, tz in p.s.electric_utility.outage_start_time_steps],
+        m[:dvMGCHPMaxFuelUsage][s] >= sum( m[:dvMGFuelUsed][t, s, tz] for t in p.techs.chp )
+    )
+    
+    @expression(m, ExpectedMGCHPFuelUsed, 
+        sum( m[:dvMGCHPMaxFuelUsage][s] * p.s.electric_utility.outage_probabilities[s] for s in p.s.electric_utility.scenarios )
+    )
+
+    # fuel cost = kWh * $/kWh
+    @expression(m, MGCHPFuelCost[t in p.techs.chp, s in p.s.electric_utility.scenarios, tz in p.s.electric_utility.outage_start_time_steps],
+        m[:dvMGFuelUsed][t, s, tz] * p.fuel_cost_per_kwh[t][tz]
+    )
+    
+    @constraint(m, [s in p.s.electric_utility.scenarios, tz in p.s.electric_utility.outage_start_time_steps],
+        m[:dvMGCHPMaxFuelCost][s] >= sum( MGCHPFuelCost[t, s, tz] for t in p.techs.chp )
+    )
+    
+    @expression(m, ExpectedMGCHPFuelCost,
+        sum( m[:dvMGCHPMaxFuelCost][s] * p.s.electric_utility.outage_probabilities[s] for s in p.s.electric_utility.scenarios )
+    )
+
+    m[:ExpectedMGFuelUsed] += ExpectedMGCHPFuelUsed
```

Comment:
How can gen and chp fuel be added together? Aren't they different units?

Reply:
Shoot yes, that's not right. It looks like we're not really using ExpectedMGFuelUsed - think we need it?

Reply:
No we just need the separate gen and CHP expected mg fuel used

Reply:
I removed the sum/total expression.